lib/babe/verify_integration_test.go

```diff
@@ -35,19 +36,21 @@ func newTestVerificationManager(t *testing.T, genCfg *types.BabeConfiguration) *
 		LogLevel:  log.Info,
 		Telemetry: telemetryMock,
 	}
+
 	dbSrv := state.NewService(config)
 	dbSrv.UseMemDB()
 
-	if genCfg == nil {
-		genCfg = genesisBABEConfig
-	}
-
 	gen, genTrie, genHeader := genesis.NewDevGenesisWithTrieAndHeader(t)
 	err := dbSrv.Initialise(gen, genHeader, genTrie)
 	require.NoError(t, err)
 
 	err = dbSrv.Start()
 	require.NoError(t, err)
+
+	if genCfg == nil {
+		genCfg = genesisBABEConfig
+	}
+
```

Comment:
why is this moved?

Reply:
if this isn't here the verifier tests panic :p